process_report/process_report.py

```diff
@@ -117,7 +147,10 @@ def main():
         help="Name of csv file listing previously billed PIs",
     )
     args = parser.parse_args()
-    merged_dataframe = merge_csv(args.csv_files)
+
+    invoice_month = args.invoice_month
+    csv_files = fetch_S3_invoices(invoice_month)
```

Comment:
Please make the upload and fetch operations togglable from a command line argument. If the options are not selected, make it work the same way with file paths.